runtime/common/src/lib.rs

```diff
@@ -805,6 +805,34 @@ macro_rules! impl_config_traits {
                     ProxyType::Staking => matches!(c, RuntimeCall::ParachainStaking(..)),
                     #[cfg(not(feature = "parachain"))]
                     ProxyType::Staking => false,
+                    ProxyType::CreateMarket =>
+                        matches!(c,
+                            RuntimeCall::PredictionMarkets(zrml_prediction_markets::Call::create_market {..}) |
```

Comment:
Market creation should also include `edit_market`.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1038/commits/b80ae75c2da1bc7ce687cb5e2a5aa6d1aef2d37c